mm2src/coins/z_coin.rs

```diff
@@ -995,7 +996,7 @@ impl<'a> ZCoinBuilder<'a> {
             priv_key_policy: PrivKeyActivationPolicy::ContextPrivKey,
             check_utxo_maturity: None,
             // This is not used for Zcoin so we just provide a default value
-            path_to_address: StandardHDCoinAddress::default(),
+            path_to_address: HDAccountAddressId::default(),
```

Comment:
I appreciate all the work for streamlining derivation path use.
As HDAccountAddressId, StandardHDPathToCoin, StandardHDPathToAccount, StandardHDPath are all part of the same 'derivation path' concept, I think maybe good to keep sharing something common in their names. How about:
HDPathToCoin
HDPathToAccount
HDPathAccountToAddressId?

Reply:
Seems good to me. Will change the name and resolve the conflicts.

Reply:
Resolved conflicts but they cause conflicts in the EVM HD wallet branch that depends on this so I will not change names until EVM HD wallet branch is merged to this to avoid more conflicts for now.

Reply:
Fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/1962/commits/a6b133df590b71efba54887aad4504a3aeb9f4e6